content/v35/3_date/page.txt

```diff
@@ -7,5 +7,139 @@ Text:
 
 (screenshot: date.png)
 
-Docs for new options are coming soon …
+Gone are the days of a date field with three select inputs with a fixed order. The new fields consist of a single text input with some sugar on tops. Which allows for a range of new possibilities - such as custom display orders and formats - but also provides a much better interaction with e.g. copying and pasting dates/times between fields.
 
+## Display in your preferred format
+
+The fields now support a new `display` option which defines how the date is displayed in the Panel. Offering a range of options:
+
+```yaml
+# default
+display: YYYY-MM-DD
+
+# For ze Germäns
+display: DD.MM.YYYY
+
+# The Brtits
```

Comment:
I'm not sure if this is a typo or intended 😂

Reply:
Oh my 😂

Reply:
Fixed